clusters-rke1/README.md

```diff
@@ -54,7 +54,7 @@ To cleanup, there is a `destroy_clusters.sh`, this script will perform a `terraf
 |------|------|
 | [null_resource.nt_dependency](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) | resource |
 | [rancher2_cluster.rke1](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/cluster) | resource |
-| [rancher2_node_pool.aws_np](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/node_pool) | resource |
+| [rancher2_node_pool.np](https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/node_pool) | resource |
```

Comment:
Just curious on the README, is this automatically generated from the variables file?

Reply:
That's neat!